src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/config/OAuth2ResourceServerSecurityConfiguration.kt

```diff
@@ -59,6 +59,7 @@ class OAuth2ResourceServerSecurityConfiguration {
         authorize(HttpMethod.DELETE, "/internal/room/*", permitAll)
         authorize(HttpMethod.GET, "/events/cas2/**", hasAuthority("ROLE_CAS2_EVENTS"))
         authorize(HttpMethod.GET, "/events/**", hasAuthority("ROLE_APPROVED_PREMISES_EVENTS"))
+        authorize(HttpMethod.PUT, "/cas2/assessments/**", hasRole("CAS2_ASSESSOR"))
```

Comment:
This role is different to the `ROLE_CAS2_ADMIN` role tested in [this commit](https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/1542/commits/8e95b58b8ff260d09f2d81f20de24530bca62d82).  Should both roles be the same?

Ref: `fun 'updating an assessment is forbidden to external users who are not Assessors'()`

Reply:
not sure what  you mean here - CAS2_ASSESSORS are able to update assessments, but CAS2_ADMINs are not.

Reply:
Resolved above - my misunderstanding.